src/ZstdSharp/Unsafe/Mem.cs

```diff
@@ -20,13 +19,13 @@ public static unsafe partial class Methods
         /*=== Static platform detection ===*/
         public static bool MEM_32bits
         {
-            [InlineMethod.Inline]
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Can we revert this for in this PR since we don't touch the `InlineMethod` yet?

Reply:
Done.